AdaptivePELE/freeEnergies/extractCoords.py

```diff
@@ -352,41 +352,49 @@ def gatherTrajs(constants, folder_name, setNumber, non_Repeat):
 def extractSidechainIndexes(trajs, ligand_resname):
     sidechains_trajs = []
     for traj in glob.glob(trajs):
-        atoms = parsePDB(traj)
+        atoms = pd.parsePDB(traj)
         sidechains = atoms.select("within 5 of resname {}".format(ligand_resname))
         sidechains_trajs.extend([atom.getIndex() for atom in sidechains])
     return list(set(sidechains_trajs))
 
 
-def main(folder_name=".", atom_Ids="", lig_resname="", numtotalSteps=0, enforceSequential_run=0, writeLigandTrajectory=True, setNumber=0, protein_CA=0, non_Repeat=False, nProcessors=None, topology=None, sidechains=False, sidechain_folder="."):
+def main(folder_name=".", atom_Ids="", lig_resname="", numtotalSteps=0, enforceSequential_run=0, writeLigandTrajectory=True, setNumber=0, protein_CA=0, non_Repeat=False, nProcessors=None, parallelize=True, topology=None, sidechains=False, sidechain_folder="."):
 
     constants = Constants()
 
     lig_resname = parseResname(atom_Ids, lig_resname)
 
+    sidechains = extractSidechainIndexes(sidechain_folder, lig_resname) if sidechains else False
+
     folderWithTrajs = folder_name
 
     makeGatheredTrajsFolder(constants)
 
     # change atomId for list
 
-    allFolders = os.listdir(folderWithTrajs)
-    folders = [epoch for epoch in allFolders if epoch.isdigit()]
-    if len(folders) == 0:
-        folders = ["."]
+
+    if enforceSequential_run:
+        allFolders = os.listdir(folderWithTrajs)
+        folders = [epoch for epoch in allFolders if epoch.isdigit()]
+        if len(folders) == 0:
+            folders = ["."]
+    else:
+        allFolders = os.listdir(folderWithTrajs)
+        folders = [epoch for epoch in allFolders if epoch.isdigit()]
+        if len(folders) == 0:
```

Comment:
This line and the next should not be here I think, looks like a copy-paste error